maven_install_2.12.json

```diff
@@ -1,6 +1,6 @@
 {
     "dependency_tree": {
-        "__AUTOGENERATED_FILE_DO_NOT_MODIFY_THIS_FILE_MANUALLY": 1241094414,
+        "__AUTOGENERATED_FILE_DO_NOT_MODIFY_THIS_FILE_MANUALLY": -1043166033,
```

Comment:
You also need to repin the scala 2.13 lockfile

Reply:
Thanks for pointing it out, it has saved me some head scratching